[Keyboard] Add Discipad, Update Zadig doc for USBaspLoader (#6771)
qmk/qmk_firmware: docs/driver_installation_zadig.md

Place your keyboard into the bootloader mode, either by hitting the `RESET` keycode (which may be on a different layer), or by pressing the reset switch usually located on the underside of the board. If your keyboard has neither, try holding Escape, or Space+`B`, as you plug it in (see the [Bootmagic](feature_bootmagic.md) docs for more details).
Some keyboards may have specific instructions for entering the bootloader, for example the [Bootmagic Lite](feature_bootmagic.md#bootmagic-lite) key (Escape) might be on a different key, such as Left Control. Refer to the board's README if you are unsure.

> To put a device in bootloader mode with USBaspLoader you must follow the following sequence: press and hold `BOOT`, press and hold `RESET`, release `RESET`, release `BOOT`.

Reviewer: :heart:

> To put a device in bootloader mode with USBaspLoader you must follow the following sequence: press and hold `BOOT`, press and hold `RESET`, release `RESET`, release `BOOT`.

Reviewer 2: ```suggestion
To put a device in bootloader mode with USBaspLoader, tap the `RESET` button while holding down the `BOOT` button.
```

Would this work too?

> An alternative method would be to press and hold `BOOT` while inserting the USB cable.

Reviewer 2: ```suggestion
Alternatively, and hold `BOOT` while inserting the USB cable.
```

Reviewer 3: I think the suggestion reads better without the `and`.

Reviewer 2: oops

> To exit bootloader mode press only the `RESET` switch.

Reviewer 2: ```suggestion
```

Author: Up to you on this one, but I put that in there because usbasploader does not automatically exit bootloader mode even after flashing, and reset must be pressed again without the boot jumper. I could mess around with fuse settings to see if I could change that, but plaid and gingham use the default fuses provided by usbasploader, so they'll be this way as well.

Reviewer 2: Most bootloaders don't exit after flashing. The Toolbox for example usually issues a reset command afterwards.

If the arrow appears green, select the driver, and click **Install Driver**. The `libusb-win32` driver will usually work for AVR, and `WinUSB` for ARM, but if you still cannot flash the board, try installing a different driver from the list.

> For flashing a USBaspLoader device via command line with msys2, the `libusbk` driver is recommended, otherwise `libusb-win32` will work fine if you are using QMK Toolbox for flashing.

Reviewer 2: This can probably go in the same paragraph.
